backend/src/authentication/mod.rs

```diff
@@ -121,28 +121,92 @@ pub async fn logout(
     Ok((updated_jar, StatusCode::OK))
 }
 
+/// Development endpoint: create a new user (unauthenticated)
+#[cfg(debug_assertions)]
```

Comment:
Maybe we want to put this behind a feature (such as `dev-database`) instead of `debug_assertions`?

Reply:
I think we want to explicitly prevent this from ever ending up in a release build. A feature flag might be enabled in a release build.